stream_alert_cli/terraform/threat_intel_downloader.py

```diff
@@ -0,0 +1,57 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from stream_alert_cli.terraform._common import DEFAULT_SNS_MONITORING_TOPIC, infinitedict
+
+def generate_threat_intel_downloader(config):
+    """Generate Threat Intel Downloader Terrafrom
+
+    Args:
+        config (dict): The loaded config from the 'conf/' directory
+
+    Returns:
+        dict: Athena dict to be marshalled to JSON
+    """
+    # Use the monitoring topic as a dead letter queue
```

Comment:
I adopted this pattern from StreamAlert App. Using Dead Letter Queue, we can indicate to Lambda that unprocessed events should be sent to SNS topic in our case. Then it will trigger CloudWatch alarm.

Reply:
@jacknagz what do you mean by 'how are they handled?' The DLQ SNS topic will receive the messages from the lambda if there is a failure, and will do whatever actions are linked to that sns topic.